.github/workflows/prettier-format.yaml

```diff
@@ -0,0 +1,21 @@
+name: Prettier check
+on:
+  pull_request:
+    branches: [main]
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.x'
+      - run: yarn install --frozen-lockfile
```

Comment:
Should we use `--immutable` here?

> If the --immutable option is set (defaults to true on CI), Yarn will abort with an error exit code if the lockfile was to be modified (other paths can be added using the immutablePatterns configuration setting). For backward compatibility we offer an alias under the name of --frozen-lockfile, but it will be removed in a later release.

https://yarnpkg.com/cli/install